src/plugins/intel_cpu/src/nodes/executors/acl/acl_deconv.hpp

```diff
@@ -0,0 +1,193 @@
+              srcDescs[1]->getPrecision() == dstDescs[0]->getPrecision())) {
+            DEBUG_LOG("AclDeconvExecutor does not support precisions:",
+                      " src[0]=", srcDescs[0]->getPrecision(),
+                      " src[1]=", srcDescs[1]->getPrecision(),
+                      " dst[0]=", dstDescs[0]->getPrecision());
+            return false;
+        }
+
+        if (deconvAttrs.withBiases && srcDescs[2]->getPrecision() != srcDescs[0]->getPrecision()) {
+            DEBUG_LOG("AclDeconvExecutor does not support precisions:",
+                      " src[2]=", srcDescs[2]->getPrecision());
+            return false;
+        }
+
+        if (!(srcDescs[0]->hasLayoutType(LayoutType::ncsp) &&
+              srcDescs[1]->hasLayoutType(LayoutType::ncsp) &&
+              dstDescs[0]->hasLayoutType(LayoutType::ncsp)) &&
+            !(srcDescs[0]->hasLayoutType(LayoutType::nspc) &&
+              srcDescs[1]->hasLayoutType(LayoutType::nspc) &&
+              dstDescs[0]->hasLayoutType(LayoutType::nspc))) {
+            DEBUG_LOG("AclDeconvExecutor does not support layouts:",
+                      " src[0]=", srcDescs[0]->serializeFormat(),
+                      " src[1]=", srcDescs[1]->serializeFormat(),
+                      " dst=", dstDescs[0]->serializeFormat());
+            return false;
+        }
+
+        if (deconvAttrs.withBiases &&
+            !(srcDescs[0]->hasLayoutType(LayoutType::ncsp) &&
+              srcDescs[1]->hasLayoutType(LayoutType::ncsp) &&
+              srcDescs[2]->hasLayoutType(LayoutType::ncsp) &&
+              dstDescs[0]->hasLayoutType(LayoutType::ncsp)) &&
+            !(srcDescs[0]->hasLayoutType(LayoutType::nspc) &&
+              srcDescs[1]->hasLayoutType(LayoutType::nspc) &&
+              srcDescs[2]->hasLayoutType(LayoutType::nspc) &&
+              dstDescs[0]->hasLayoutType(LayoutType::nspc))) {
+            DEBUG_LOG("AclDeconvExecutor does not support layouts:",
+                      " src[0]=", srcDescs[0]->serializeFormat(),
+                      " src[1]=", srcDescs[1]->serializeFormat(),
+                      " src[2]=", srcDescs[2]->serializeFormat(),
+                      " dst=", dstDescs[0]->serializeFormat());
+            return false;
+        }
+
+        auto srcDims  = srcDescs[0]->getShape().getDims();
+        auto weiDims  = srcDescs[1]->getShape().getDims();
+        // swap input and output channels dimensions to be align with ACL
+        // weights tensor shape is changed because ACL expects [W, H, I, O] tensor while OV uses [I, O, H, W] tensor
+        std::swap(weiDims[0], weiDims[1]);
+        auto dstDims  = dstDescs[0]->getShape().getDims();
+
+        VectorDims biasDims;
+        arm_compute::TensorInfo biasTensorInfo;
+        if (deconvAttrs.withBiases) {
+            biasDims = srcDescs[2]->getShape().getStaticDims();
+            //bias presicion is I32 but ACL requests bias precision as input ones
```

Comment:
if executor supports only equal precision for bias you have to preperly set precisions on Deconv node level (supportedPrimitiveDescriptors initialization code)

Reply:
@dmitry-gorokhov corrected